libUtils/windowsChromeDriver.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import requests
+import os
+import zipfile
+
+class BrowserManager:
+    def __init__(self,extract_dir=".//dependencies//chrome-driver"):
+        self.extract_dir = extract_dir
+    def get_chrome_version(self):   #    Get the Chrome browser version installed on Windows.
+        try:
+            output = subprocess.check_output(
+                r'reg query "HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version',
+                shell=True,
+                encoding='utf-8'
+            )
+            # Extract version from the command output
+            version = output.split()[-1]
+            print(f"Chrome Version: {version}")
+            return version
+
+        except Exception as e:
+            raise Exception(f"Error getting Chrome version: {e}")
+
+    def get_chrome_driver_download_url(self, chrome_version):
+        try:
+            if not chrome_version:
+                raise Exception("Chrome version is required to fetch the download URL.")
+
+            download_url = f"https://storage.googleapis.com/chrome-for-testing-public/{chrome_version}/win64/chromedriver-win64.zip"
```

Comment:
moved the base url to inside init method like self.driver_base_url  =f"https://storage.googleapis.com/chrome-for-testing-public/"  and  use it here